cepheus-cep/src/test/java/com/orange/cepheus/cep/SubscriptionManagerTest.java

```diff
@@ -47,7 +62,8 @@
 @SpringApplicationConfiguration(classes = Application.class)
 @DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
 public class SubscriptionManagerTest {
-
+    static boolean flag=true;
```

Comment:
This is not a clean way to pass state to `getBasicConf()`, add an explicit parameter (like `Bool addServiceNameAndPath) to `getBasicConf()` or duplicate this method and create a `getBasicConfWithServiceNameAndPath()`

Reply:
According to my understanding i need to change the name of "flag" to "addServiceNameAndPath".
please correct me if i am wrong.

Reply:
Please don't use static variables where you can pass a simple argument to a method (the case here).
Add a `addServiceNameAndPath` argument to getBasicConf() or create a new method getBasicConfWithService() that calls getBasicConf() and appends a serviceName/Path to the configuration.